tools/demo/index.css

```diff
@@ -47,4 +47,10 @@ body {
 }
 .dg .function .property-name {
     width: 100%;
-}
\ No newline at end of file
+}
+.dg .property-name {
+    width: 46%;
+}
+.dg .c {
+    width: 54%;
+}
```

Comment:
What is the purpose of these styles?

Reply:
If we don't make the first column become bigger , we can't see the full content of some properties's name